media-sound/audacious/audacious-4.4.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_P="${P/_/-}"
+
+inherit meson xdg
+
+DESCRIPTION="Lightweight and versatile audio player"
+HOMEPAGE="https://audacious-media-player.org/"
+SRC_URI="
+	https://distfiles.audacious-media-player.org/${MY_P}.tar.bz2
+	mirror://gentoo/gentoo_ice-xmms-0.2.tar.bz2
+"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+
+BDEPEND="
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-libs/glib:2
+	dev-qt/qtbase:6=[gui,widgets]
+	dev-qt/qtsvg:6=
```

Comment:
Does it use private headers for both? Otherwise the slot op is bogus.

Reply:
Ah, thanks. I suspect not, but I will check.

Reply:
Yeah, I see no evidence of using private Qt headers, so I've dropped the `=`.